src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
+
+const SPINNER_BASE_CLASS = 'spinner';
+
+const SPINNER_INVERSE = 'spinner-inverse';
+const SPINNER_INLINE = 'spinner-inline';
+
+const SPINNER_SMALL_SIZE = 'spinner-sm';
+const SPINNER_MEDIUM_SIZE = 'spinner-md';
+
+@Component({
+  selector: 'clr-spinner',
+  template: `
+    <ng-content></ng-content>
+  `,
+  host: {
+    '[attr.class]': 'assignClass',
```

Comment:
We cannot use `attr.class` as it overrides the entire value of the class attribute. That means the classes defined by a user will disappear. We need to use `@HostBinding`.